webapp/src/dialogs.tsx

```diff
@@ -689,6 +689,25 @@ export function showReportAbuseAsync(pubId?: string) {
     })
 }
 
+export function showWinAppDeprecateAsync() {
+    pxt.tickEvent("winApp.dialog", undefined)
+    return core.confirmAsync({
+        header: lf("You can't get there from here!"),
```

Comment:
oh also wasn't sure about this header text. stole this from the dialog i see anytime i try to log into teams on an illegal device 🙂